versioned_docs/version-1.5/reference/manifest.mdx

```diff
@@ -214,16 +223,34 @@ deploy:
   compose: docker-compose.yml
 ```
 
-You can share environment variables between steps by adding them to `$OKTETO_ENV`:
+Your deploy commands will be executed before deploying your Docker compose files.
+
+#### Divert
+
+There is an extended notation to divert a namespace as part of your development environment:
 
 ```yaml
 deploy:
-  - name: Set env value
-    command: echo "OKTETO_FOLDER=/app" >> $OKTETO_ENV 
-  - name: Get env value
-    command: echo "$OKTETO_FOLDER" # Prints /app
+  commands:
+    - helm upgrade --install movies chart --set api.image=${OKTETO_BUILD_API_IMAGE} --set frontend.image=${OKTETO_BUILD_FRONTEND_IMAGE}
+  divert:
+    driver: istio
+    namespace: staging
+    service: api
+    virtualServices:
+      - frontend
 ```
 
+When divert is enabled, Okteto will add the header `x-okteto-divert` to every request coming from the developer namespace.
```

Comment:
Can you put the 3 paragraphs of explanation before the sample? I think it reads better that way.